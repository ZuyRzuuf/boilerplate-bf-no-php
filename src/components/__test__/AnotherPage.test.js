import * as React from 'react';

import AnotherPage from '../AnotherPage';

describe('<AnotherPage /> suite', function() {
    it('renders correctly isolated component', () => {
        const wrapper = shallow(
            <AnotherPage />
        );
        expect(wrapper).toMatchSnapshot();
    });

    it('renders correctly HTML tree', () => {
        const wrapper = render(
            <AnotherPage />
        );
        expect(wrapper).toMatchSnapshot();
    });

    it('renders correctly components tree', () => {
        const wrapper = mount(
            <AnotherPage />
        );
        expect(wrapper).toMatchSnapshot();
    });
});
