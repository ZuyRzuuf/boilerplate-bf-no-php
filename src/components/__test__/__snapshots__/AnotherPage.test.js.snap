// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<AnotherPage /> suite renders correctly HTML tree 1`] = `
<div>
  AnotherPage
</div>
`;

exports[`<AnotherPage /> suite renders correctly components tree 1`] = `
<AnotherPage>
  <div>
    AnotherPage
  </div>
</AnotherPage>
`;

exports[`<AnotherPage /> suite renders correctly isolated component 1`] = `
<Fragment>
  <div>
    AnotherPage
  </div>
</Fragment>
`;
